hypothesis-python/src/hypothesis/_settings.py

```diff
@@ -768,7 +771,7 @@ def note_deprecation(
 
 
 # This is tested in a subprocess so the branch doesn't show up in coverage.
-if is_in_ci():  # pragma: no cover
+if is_in_ci():  # pragma: no
```

Comment:
typo?

Reply:
Yes. Also apparently unneeded given that coverage tests aren't failing!